include/ignition/math/MassMatrix3.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
this file should be deleted

Reply:
Noting: git picked up the math.hh deletion and Export creation as a move for some reason.